frontend/src/utils/YupSchema.tsx

```diff
@@ -50,7 +50,7 @@ export const Address = Yup.object().shape({
     .nullable(),
   provinceId: Yup.string().required('Required'),
   postal: Yup.string().matches(
-    /^[a-zA-z][0-9][a-zA-z]\s*?[0-9][a-zA-z][0-9]$/,
+    /^[a-zA-z][0-9][a-zA-z]\s*-*[0-9][a-zA-z][0-9]$/,
```

Comment:
Odd regex for the center space.  You're saying 0 or more spaces or 0 or more dashes.  I'm fairly certain you just want [\s-]?

Reply:
Oh woops yes you're right